requirements/edx/github.txt

```diff
@@ -35,10 +35,10 @@ git+https://github.com/pmitros/pyfs.git@96e1922348bfe6d99201b9512a9ed946c87b7e0b
 -e git+https://github.com/edx/bok-choy.git@82d2f4b72e807b10d112179c0a4abd810a001b82#egg=bok_choy
 -e git+https://github.com/edx-solutions/django-splash.git@7579d052afcf474ece1239153cffe1c89935bc4f#egg=django-splash
 -e git+https://github.com/edx/acid-block.git@e46f9cda8a03e121a00c7e347084d142d22ebfb7#egg=acid-xblock
--e git+https://github.com/edx/edx-ora2.git@release-2015-04-15T15.17#egg=edx-ora2
+-e git+https://github.com/singingwolfboy/edx-ora2.git@update-pytz#egg=edx-ora2
```

Comment:
We should not be using your fork of edx-ora2. Let's get the fix into edx-ora2 instead, since you already have PR on that repo already.